skeleton/cli-bundler/aurelia_project/tasks__if_babel/transpile.js

```diff
@@ -49,13 +62,30 @@ export function buildPluginJavaScript(dest, format) {
     if (format === 'commonjs') {
       opts = {
         plugins: [
-          ['@babel/plugin-transform-modules-commonjs', {loose: true}]
+          ['@babel/plugin-transform-modules-commonjs', {
+            loose: true
+          }]
         ]
       };
     }
     const transpile = babel(opts);
+
+    let defaultBuildOptions = {
+      minify: 'stage & prod',
+      sourcemaps: 'dev & stage',
+      rev: false
+    };
+
+    let buildOptions = new Configuration(project.build.options, defaultBuildOptions);
+    let sourcemaps = buildOptions.isApplicable('sourcemaps');
```

Comment:
This seems unnecessary. Why not just turn on sourcemap without condition? That's what core aurelia modules are trying to do now, shipping sourcemap in all the builds.

It's app bundler's job to decide whether to use the source map in dev build, or strip the source map in production build. All bundlers are capable of doing that.